DQMOffline/Trigger/python/HTMonitor_cff.py

```diff
@@ -0,0 +1,232 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+
+# HLT_PFMETNoMu90_PFMHTNoMu90_IDTight
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring = hltHTmonitoring.clone()
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring.FolderName = cms.string('HLT/HT/PFMETNoMu90/')
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_PFMETNoMu90_PFMHTNoMu90_IDTight_v*")
+
+# HLT_PFMETNoMu120_PFMHTNoMu120_IDTight
+PFMETNoMu120_PFMHTNoMu120_HTmonitoring = hltHTmonitoring.clone()
+PFMETNoMu120_PFMHTNoMu120_HTmonitoring.FolderName = cms.string('HLT/HT/PFMETNoMu120/')
+PFMETNoMu120_PFMHTNoMu120_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_v*")
+
+# HLT_MET200
+MET200_HTmonitoring = hltHTmonitoring.clone()
+MET200_HTmonitoring.FolderName = cms.string('HLT/HT/MET200/')
+MET200_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_MET200_v*")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_IDTight
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu110/')
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_IDTight_v*")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.jetSelection      = cms.string("pt[0] > 100 && eta[0] < 2.5 && neutralHadronEnergyFraction[0] < 0.8 && chargedHadronEnergyFraction[0] > 0.1")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.muoSelection      = cms.string("isTightMuon()")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.jetSelection_HT   = cms.string("pt > 30 && eta < 2.5")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_IDTight
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu120/')
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_IDTight_v*")
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.jetSelection_HT   = cms.string("pt > 30 && eta < 2.5")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_IDTight
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu130/')
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_IDTight_v*")
```

Comment:
please fix the path name to `HLT_MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_IDTight_v*`